plugins/woocommerce/includes/gateways/paypal/includes/class-wc-gateway-paypal-request.php

```diff
@@ -799,6 +879,6 @@ protected function number_format( $price, $order ) {
 			$decimals = 0;
 		}
 
-		return number_format( $price, $decimals, '.', '' );
+		return number_format( (float) $price, $decimals, '.', '' );
```

Comment:
This is unrelated to the actual capture payment flow.
As I included `declare(strict_types=1);` to fix a lint error, a couple of tests started failing in this line because of the strict type check.